src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,188 @@
+          default:
+            return 0;
+        }
+      },
+    );
+
+    if (order === 'desc') {
+      sorted.reverse();
+    }
+
+    return sorted;
+  }, [filteredPeople, sort, order]);
+
+  const toggleOrder = (sortName: string) => {
+    if (!sort) {
+      return { sort: sortName, order: null };
+    }
+
+    if (sortName && !order) {
+      return { sort: sortName, order: 'desc' };
+    }
+
+    if (sortName && order) {
+      return { sort: null, order: null };
+    }
+
+    if (!sortName && !order) {
+      return { sort: null, order: null };
+    }
+
+    return { sort: sortName, order: null };
+  };
+
+  return !sortedPeople.length ? (
+    <p>There are no people matching the current search criteria</p>
+  ) : (
     <table
       data-cy="peopleTable"
       className="table is-striped is-hoverable is-narrow is-fullwidth"
     >
       <thead>
         <tr>
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Name
-              <a href="#/people?sort=name">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Sex
-              <a href="#/people?sort=sex">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Born
-              <a href="#/people?sort=born&amp;order=desc">
-                <span className="icon">
-                  <i className="fas fa-sort-up" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Died
-              <a href="#/people?sort=died">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
-            </span>
-          </th>
+          {sortedOptions.map(option => (
+            <th key={option}>
+              <span className="is-flex is-flex-wrap-nowrap">
+                {option}
+                <SearchLink params={toggleOrder(option.toLowerCase())}>
+                  <span className="icon">
+                    <i
+                      className={classNames('fas', {
+                        'fa-sort': sort !== option.toLowerCase(),
+                        'fa-sort-up': sort === option.toLowerCase() && !order,
+                        'fa-sort-down':
+                          sort === option.toLowerCase() && order === 'desc',
+                      })}
+                    />
+                  </span>
+                </SearchLink>
+              </span>
+            </th>
+          ))}
```

Comment:
The table headers are hardcoded. Consider making them dynamic so that if you add more fields to the `Person` type, you won't have to manually update the table headers.